LibTest/js_interop/JSExport/jsExport_A06_t05.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Only concrete instance members can and will be wrapped, and it's
+/// an error to annotate other members with this annotation.
+///
+/// @description Checks that alias of a class annotated with `@JSExport()` also
+/// can be annotated with `@JSExport()`.
+/// @author sgrekhov22@gmail.com
+
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
+import '../../../Utils/expect.dart';
+import '../js_utils.dart';
+
+String log = "";
+
+@JSExport()
+class C {
+  int variable = 42;
+  String method(String v) => "method($v);";
+  String get getter => "Some getter";
+  void set setter(bool value) {
+    log = "setter($value);";
+  }
+}
+
+@JSExport()
```

Comment:
This could be a warning so users don't think it makes a difference, but I don't even know if this declaration appears in the CFE, so we might not be able to bubble up a warning here.

IIRC, typedefs are resolved early, so we would never even see `CAlias` as a type in the kernel.